wasm-engines/Dockerfile

```
@@ -3,8 +3,24 @@ FROM ubuntu:19.10
 # System deps
 RUN apt-get clean
 RUN apt-get update
-RUN apt-get install -y software-properties-common git sudo build-essential wget curl nano \
-    autoconf automake libtool llvm-6.0 make ninja-build unzip zlib1g-dev texinfo libssl-dev
+RUN apt-get install -y software-properties-common gnupg1 cmake gcc-8 g++-8 \
+        libboost-all-dev git sudo build-essential wget curl \
+        ninja-build libssl-dev
+
+WORKDIR /engines
+
+RUN wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key|sudo apt-key add -
+RUN apt update && apt install -y \
+	libllvm9 \
+	llvm-9 \
+	llvm-9-dev \
+	llvm-9-runtime \
+	libclang-common-9-dev # for yaml-bench
+
+RUN git clone https://github.com/hugo-dc/SSVM.git --single-branch --branch bench
```

Cannot this be used?

```suggestion
RUN git clone https://github.com/ewasm-benchmarking/SSVM.git --single-branch --branch bench
```

---

I was planning to create a different PR updating the docker file to use the ewasm-benchmarking repositories, but you're right it makes sense to already make use of the ewasm-benchmarking/SSVM for.

---

I'm fine with either way, its up to you.

---

I updated the Dockerfile to use the ewasm-benchmarking fork.